src/main/java/edu/ie3/datamodel/io/factory/Factory.java

```diff
@@ -48,15 +49,10 @@ public List<Class<? extends C>> getSupportedClasses() {
   public Try<R, FactoryException> get(D data) {
     isSupportedClass(data.getTargetClass());
 
-    // magic: case-insensitive get/set calls on set strings
-    final List<Set<String>> allFields = getFields(data);
-
     try {
-      validateParameters(data, allFields.toArray((IntFunction<Set<String>[]>) Set[]::new));
-
       // build the model
       return Success.of(buildModel(data));
-    } catch (FactoryException e) {
+    } catch (Exception e) {
```

Comment:
Inside the `buildModel` method other exceptions like `IllegalArgumentException` can be thrown. These exceptions need to be catched, because otherwise no `Try` is returned. A possible solution would be to catch all types of exceptions that coulb be thrown.